docs/reference/online-stores/README.md

```diff
@@ -46,6 +46,10 @@ Please see [Online Store](../../getting-started/components/online-store.md) for
 [cassandra.md](cassandra.md)
 {% endcontent-ref %}
 
+{% content-ref url="couchbase.md" %}
+[couchbase.md](cassandra.md)
```

Comment:
This days Cassandra, is that a typo?

Reply:
Ah good catch, thanks. Fixed: https://github.com/feast-dev/feast/pull/4637/commits/d1ae5d9aa65b2204a1c4aa93d789a9a20a7ab7d5